src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj

```diff
@@ -23,7 +23,8 @@
     <PackageReleaseNotes>
 - Use convert setting to toggle between referencing @odata.count and @odata.nextLink #282
 - Fixes URL Path parameters of type datetime generated as strings with quotes #262
-- Add support for alternate keys #120
+- Generate odata.nextLink and odata.deltaLink properties on delta functions response schemas #285
+- Add support for alternate key parameters#120
     </PackageReleaseNotes>
     <AssemblyName>Microsoft.OpenApi.OData.Reader</AssemblyName>
```

Comment:
once you rebase and handle conflicts + feedback, can you bump to preview 4 please?